app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="container">
+  <h2 class="ang">Log in</h2>
```

Comment:
- [ ] This activity requires the modification of the views in devise for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly. Kindly ensure to improve upon the same.

**Your login view:**
![image](https://user-images.githubusercontent.com/12745474/201101020-fc9a271b-ba13-4632-984d-ea5950eb4598.png)

**Wireframe login view:**
![image](https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png)